projects/domain-events-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/entity/Nsi.kt

```diff
@@ -0,0 +1,65 @@
+package uk.gov.justice.digital.hmpps.integrations.delius.entity
+
+import org.hibernate.annotations.Immutable
+import java.time.LocalDate
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.Table
+
+@Immutable
+@Entity
+class Nsi(
+    @Id
+    @Column(name = "nsi_id")
+    var id: Long,
+
+    val referralDate: LocalDate,
```

Comment:
We try to use ZonedDateTime where possible, is there a reason for using LocalDate here?  Same in the `BreachDetails` response.